src/main/java/frc/robot/commands/shooter/ShootThreeBalls.java

```diff
@@ -0,0 +1,42 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands.shooter;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.ShooterRoutine;
+
+import frc.robot.Settings.Constants;
+import frc.robot.Settings.Variables;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.Indexer;
+import frc.robot.subsystems.Shooter;
+import frc.robot.commands.timeDrive;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class ShootThreeBalls extends SequentialCommandGroup {
+  private Shooter shooter;
+  private Indexer indexer;
+  private Drivetrain drivetrain;
+  private ShooterRoutine shooterRoutine;
+  private timeDrive timeDrive;
+  /**
+   * Creates a new ShootThreeBalls.
+   */
+  public ShootThreeBalls(Drivetrain drivetrain, Shooter shooter, Indexer indexer, Drivetrain driveTransfer, double speedDriveTransfer, double timeTransfer) {
+    this.drivetrain = drivetrain;
+    this.indexer = indexer;
+    this.shooter = shooter;
+    // Add your commands in the super() call, e.g.
+    // super(new FooCommand(), new BarCommand());
+    addCommands(
+      new ShooterRoutine(shooter, indexer),
```

Comment:
There are man changes that have to be made here. 

- These need to be sequential not at the same time.
- Make sure that the shooter routine is really doing what you think it's doing.

Reply:
hey i think it works